README.md

```diff
@@ -12,7 +12,7 @@ and [Source](https://docs.opn.ooo/sources-api) API and components for entering c
 
 ## Security Warning
 
-**Please do NOT use Omise Android SDK versions less than 3.0.0, as they are outdated and have security vulnerabilities.**
+**It is imperative that you use at least the minimum recommended SDK version of `4.3.1` for security reasons. Any version below this poses severe risks of security vulnerabilities, bugs, and unexpected behaviors, which can be detrimental to your application. To avoid these risks and ensure the best user experience, it is highly recommended that you upgrade to the latest supported SDK version. Opn strongly advises the use of version `5.0.0` for superior performance and top-notch security. Don't compromise on the security and performance of your application - upgrade to the latest SDK version today.**
```

Comment:
`upgrade to the latest SDK version today`
@muthuswamyopn  This sentence looks a bit weird as if its an advertisement.